requirements.txt

```diff
@@ -0,0 +1 @@
+pyalsaaudio  # for speechRecognition.py
```

Comment:
Missing packages for gi/gtk/gobject (might be related). Just noticing that I had to install the `python-gi` apt package where a pip one would be preferable.

Reply:
Now commented at https://github.com/roboticslab-uc3m/speech/blob/41336b99e01fad4c8ad9244dd2c7fe18fc9182ee/doc/speech-install.md#troubleshooting-installation (was not required for me, on a clean Xenial + pip2)